examples/connect/server.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+// eslint-disable-next-line
+const tracing = require('./tracing')('example-connect-server');
+
+// Require in rest of modules
+const connect = require('connect');
+const axios = require('axios');
+
+// Setup express
```

Comment:
```suggestion
// Setup connect
```
In the example package name I'd actually leave express there(or some other kind of specifier, since the package name is so generic).